java/sage/EPG.java

```diff
@@ -597,6 +598,12 @@ public java.io.File getLogoDir()
 
   public void run()
   {
+
+    //03-02-2025 jusjoken: added a version check here.  As this EPG process runs basically daily I added the
+    //  version check here.  It may fit best elsewhere but I will leave it here for now to ensure future updates are noticed
+    //check if an updated version is available on github
+    wiz.checkForUpdate();
```

Comment:
You should probably remove this, and just leave the one below that happens in the loop. It's generally not a great idea to do some kind of network check during startup unless it's critical (and this is just informative where it creates a system message).

Reply:
Ok.  Done and will be in a new commit